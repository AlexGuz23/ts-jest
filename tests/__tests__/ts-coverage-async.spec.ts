import runJest from '../__helpers__/runJest';

describe('Typescript async coverage', () => {
  it('Should generate the correct async coverage numbers', () => {
    const result = runJest('../simple-async', ['--no-cache', '--coverage']);

    const output = result.stdout;

    expect(output).toMatchSnapshot();
  });
});
