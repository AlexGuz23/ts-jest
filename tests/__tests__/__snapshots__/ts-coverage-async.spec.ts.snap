// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Typescript async coverage Should generate the correct async coverage numbers 1`] = `
"
=============================== Coverage summary ===============================
Statements   : 71.43% ( 10/14 )
Branches     : 33.33% ( 2/6 )
Functions    : 66.67% ( 4/6 )
Lines        : 66.67% ( 8/12 )
================================================================================
"
`;
